.github/workflows/optimize-create-release-branch.yml

```diff
@@ -19,48 +16,19 @@ jobs:
     strategy:
       fail-fast: true
     steps:
-      - name: Checkout
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
-
-      - name: Import Secrets
-        id: secrets
-        uses: hashicorp/vault-action@d1720f055e0635fd932a1d2a48f87a666a57906c # v3.0.0
-        with:
-          url: ${{ secrets.VAULT_ADDR }}
-          method: approle
-          roleId: ${{ secrets.VAULT_ROLE_ID }}
-          secretId: ${{ secrets.VAULT_SECRET_ID }}
-          secrets: |
-            secret/data/products/optimize/ci/camunda-optimize GITHUB_OPTIMIZE_APP_ID;
-            secret/data/products/optimize/ci/camunda-optimize GITHUB_OPTIMIZE_APP_KEY;
-
-      - name: Generate a GitHub token
-        id: github-token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2
-        with:
-          app_id: ${{ steps.secrets.outputs.GITHUB_OPTIMIZE_APP_ID }}
-          private_key: ${{ steps.secrets.outputs.GITHUB_OPTIMIZE_APP_KEY }}
-
-      - name: Setup Maven
-        uses: ./.github/actions/setup-maven
+      - name: Checkout release branch
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
         with:
-          secrets: ${{ toJSON(secrets) }}
+          ref: ${{ github.event.inputs.release_base_branch }}
+          fetch-depth: 0
 
-      - name: Get next development version
-        id: next_development_version
-        run: ./.github/optimize/scripts/get-next-development-version.sh
-        env:
-          RELEASE_TYPE: ${{ github.event.inputs.release_type}}
-          RELEASE_VERSION: ${{ github.event.inputs.release_version}}
-          RELEASE_ALPHA_NUMBER: ${{ github.event.inputs.release_alpha_number}}
+      - name: Configure GitHub user
+        run: |
+          git config --global user.name "optimize-release"
+          git config --global user.email "ci@optimize.camunda.cloud"
 
       - name: Create release branch
-        run: ./.github/optimize/scripts/create-release-branch.sh
-        env:
-          GITHUB_APP_PRIVATE_KEY: ${{ steps.github-token.outputs.token }}
-          GITHUB_ACTOR_ID: ${{ github.actor_id }}
-          GITHUB_ACTOR: ${{ github.actor }}
-          GITHUB_TOKEN: ${{ steps.github-token.outputs.token }}
-          RELEASE_TYPE: ${{ github.event.inputs.release_type}}
-          RELEASE_VERSION: ${{ github.event.inputs.release_version}}
-          NEXT_DEVELOPMENT_VERSION: ${{ steps.next_development_version.outputs.next_development_version }}
```

Comment:
This code will update the development version in the pom.xml
For exmple, master is on 8.x.x-alpha1 and we are trying to release alpha2
This will update the pom to the new version (8.x.x-alpha2)

How do we plan to do this with the new workflow?

Reply:
we are not gonna do this, we are only relying on the `release:prepare` and `release:perform` commands from the release workflow. I was taking inspiration from Operate. They are not using `gitflow` at all and for us it was only bringing confusion and failing all the time, also it was using a third branch called `latest`. With the current approach it will all stay as it is on `main` and all the version bumps are gonna be done on the release branch only, so `main` won't break and when we are done with the release we will simply open a PR from `release` branch to `main`.
Does it make sense?

Reply:
Okay, sounds good 👍